tests/test_variant_util.py

```diff
@@ -310,6 +308,19 @@ def test_retrieve_variants_of_region(
         assert last.start == 22_130_691
         assert last.end == 22_130_692
 
+    def test_get_allele_frequency(
```

Comment:
Assuming the issues within `get_allele_frequency` body I commented on today are real bugs, you really need a better test suite, because the current did not test for the bugs!